layouts/_default/pricing.html

```diff
@@ -0,0 +1,330 @@
+                                        fill="#222222" />
+                                </svg>
+                            </div>
+                            <div>
+                                <p class="text-[#222222] mb-0">Starter</p>
+                            </div>
+                        </div>
+                        <div>
+                            <header class="text-center h-[80px] lg:h-[100px] xl:h-[80px]">
+                                <h3 class="text-[#222222] text-lg font-bold">For individual use</h3>
+                            </header>
+                            <div class="h-[280px] lg:h-[350px] xl:h-[280px] border-b border-dashed border-[#D9D9D9]">
+                                <ul class="pl-0">
+                                    <li class="flex mb-2">
+                                        <div class="flex-none w-[20px]">
+                                            <svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                                <path d="M6 12.0001L10.243 16.2431L18.727 7.75708" stroke="#444083"
+                                                    stroke-width="3" stroke-linecap="round" stroke-linejoin="round" />
+                                            </svg>
+                                        </div>
+                                        <p class="text-[#222222] text-sm mb-0 ml-2">Run local code in the context of
+                                            your
+                                            cloud environment!</p>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                        <div>
+                            <div class="text-center pt-6 h-[125px] lg:h-[140px] xl:h-[125px]">
+                                <h2 class="text-[#222222] text-3xl lg:text-4xl font-medium mb-2">$0</h2>
+                                <h3 class="text-[#222222] font-medium mb-0">Free and open-source</h3>
+                            </div>
+                            <div class="flex justify-center px-4 mb-6">
+                                <a class="w-full h-[46px] lg:h-[52px] flex items-center justify-center text-white bg-primary border border-solid border-[#222222] rounded-[6px] lg:rounded-[10px] shadow-[-7px_6px_0px_#000000] lg:shadow-[-9px_8px_0px_#000000] transition-colors duration-200 ease-[ease]"
+                                    href="/">Get Started</a>

```

Comment:
```suggestion
                                    href="/docs/overview/quick-start/">Get Started</a>

```

Maybe link to quick-start